langchain4j-core/src/test/java/dev/langchain4j/data/document/MetadataTest.java

```diff
@@ -120,7 +120,7 @@ public void test_remove() {
     void test_asMap() {
         Metadata metadata = Metadata.from("key", "value");
 
-        Map<String, String> map = metadata.asMap();
+        Map<String, Object> map = metadata.toMap();
```

Comment:
I think that `MetadataTest` has more deprecated method usages

Reply:
Thank you, yes, I fixed them too.